rm/rm-server/src/test/java/org/ow2/proactive/resourcemanager/db/NodeSourcesTest.java

```diff
@@ -43,13 +46,22 @@
 
 public class NodeSourcesTest extends ProActiveTest {
 
+    // Arbitrary values used for the test
+    private static final String INFRASTRUCTURE_VARIABLE_KEY = "key";
+
+    private static final String INFRASTRUCTURE_VARIABLE_VALUE = "value";
+
     protected static RMDBManager dbManager;
 
+    private Map<String, Object> infrastructureVariables;
+
     @Before
     public void setUp() throws Exception {
         PAResourceManagerProperties.RM_ALIVE_EVENT_FREQUENCY.updateProperty("100");
         Configuration config = new Configuration().configure("/functionaltests/config/hibernate.cfg.xml");
         dbManager = new RMDBManager(config, true, true);
+        infrastructureVariables = new HashMap<>();
+        infrastructureVariables.put(INFRASTRUCTURE_VARIABLE_KEY, INFRASTRUCTURE_VARIABLE_VALUE);
     }
 
     @After
```

Comment:
I think it is useful in order to release the dbManager object properly after each test.